plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/util/MavenUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.tools.eclipse.util;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+
+import com.google.cloud.tools.eclipse.appengine.localserver.Activator;
+
+public class MavenUtils {
+
+  public static final String OLD_MAVEN2_NATURE_ID = "org.maven.ide.eclipse.maven2Nature";
+  public static final String MAVEN2_NATURE_ID = "org.eclipse.m2e.core.maven2Nature";
+
+  /**
+   * Returns <code>true</code> if the given project has the Maven 2 nature. This
+   * checks for the old maven nature (till m2Eclipse 0.12) and the new Maven
```

Comment:
0.12 was the last version of the plugin from Sonatype. Since then it is released by Eclipse and uses a new ID for the nature. It is at version 1.6.2, which is also included in the `m2Eclipse 1.0.0 and up` phrase so I think we're good here.